docs/guide/subscriptions.md

```diff
@@ -1,6 +1,140 @@
-### Subscriptions
+# Subscriptions
 
 Subscriptions are supported using the
 [Strawberry Django Channels](https://strawberry.rocks/docs/integrations/channels) integration.
 
-Check its docs to know how to use it.
+This guide will give you a minimal working example to get you going.
+There are 3 parts to this guide:
+
+1. Making Django compatible
+2. Setup local testing
+3. Creating your first subscription
+
+## Making Django compatible
+
+It's important to realise that Django doesn't support websockets out of the box.
+To resolve this, we can help the platform along a little.
+
+This implementation is based on Django Channels - this means that should you wish - there is a lot more websockets fun to be had. If you're interested, head over to [Django Channels](https://channels.readthedocs.io).
+
+To add the base compatibility, go to your `MyProject.asgi.py` file and replace it with the following content.
+Ensure that you replace the relevant code with your setup.
+
+```python
+# MyProject.asgi.py
+import os
+
+from django.core.asgi import get_asgi_application
+from strawberry_django.routers import AuthGraphQLProtocolTypeRouter
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "MyProject.settings")  # CHANGE the project name
+django_asgi_app = get_asgi_application()
+
+# Import your Strawberry schema after creating the django ASGI application
+# This ensures django.setup() has been called before any ORM models are imported
+# for the schema.
+
+from .schema import schema  # CHANGE path to where you housed your schema file.
+application = AuthGraphQLProtocolTypeRouter(
+    schema,
+    django_application=django_asgi_app,
+)
+```
+
+Also, ensure that you enable subscriptions on your AsgiGraphQLView in `MyProject.urls.py`:
+
+```python
+...
+
+urlpatterns = [
+	...
+    path(
+        'graphql/',
+        AsyncGraphQLView.as_view(
+            schema=schema,
+            graphiql=settings.DEBUG,
+            subscriptions_enabled=True,
+        ),
+    ),
+    ...
+]
+
+```
+
+Note, django-channels allows for a lot more complexity. Here we just cover the basic framework to get
```

Comment:
obs. pre-commit alex won't allow the word `just` to be used. It might need to be reworded here.

https://results.pre-commit.ci/run/github/308695886/1696869627.klCewzb2Qp6uMthM-uq5iw

Reply:
Thanks for the help.  I've adjusted the wording.  
I did add a get_current_user override to allow for quick testing earlier today.  If you like what I did, let me know and I'll add it to the docs as well.